configs/datamodule/moons.yaml

```diff
@@ -1,8 +1,20 @@
 _target_: src.datamodules.sklearn_datamodule.create_sklearn_datamodule
-dataset:
+
+train_dataset:
```

Comment:
Since Train, Val, and Test datasets are initialized separately using `sklearn.datasets.make_moons`, there could be data leakage across the splits. I am working on a potential fix for this, perhaps we can discuss more in the meeting.